### DISCUSSION TITLE
How i can play video through arraylist

### DISCUSSION THREAD

**Discussion by nitish8879** [moneytoo/Player#110] [Q&A]
I want to play videos my internal storage in your PLayerActivity when I try to send from my ArrayList position to your Player Activity it'll just crash the app I don't know what's the exact problem. can anybody help me

**Comment by moneytoo** [comment]
This player isn't really made for being used in other apps. I suggest you using ExoPlayer directly ([playlists](https://exoplayer.dev/playlists.html)).
